config/build_conf.yaml

```diff
@@ -10,7 +10,7 @@ project:
   title: "PHP Library Manual"
   branched: true
 version:
-  release: '1.6'
+  release: '1.7'
```

Comment:
Is this the default version?

I'm curious if this means that the `master` (upcoming) branch will also be reported as 1.7. Now that we have a v1.7 branch in PHPLIB, the `master` branch is technically `1.8-dev`.

Reply:
I think it will need an update for master branch, will look at what other repos are doing.

Reply:
Confirmed with the docs platform team that this is a label, so I'll create a separate PR for master to show 1.8-dev